index.js

```diff
@@ -103,54 +104,64 @@ module.exports = (robot) => {
           // eslint-disable-next-line no-console
           console.log('No data found.');
         } else {
-          for (var i = 0; i < rows.length; i++) {
-            var rowUserName = rows[i][0];
-            if (rowUserName === userName) {
-              hasUserSignedCla = true;
-              break;
+          var params;
+
+          const labels = context.github.issues.getIssueLabels(
+            context.issue());
+          var labelData, claFlag = false;

```

Comment:
For the interest of readability, separate this into two lines.

Reply:
Done, thanks!